libcontainer/capabilities/capabilities.go

```diff
@@ -25,19 +25,23 @@ var (
 )
 
 func init() {
-	capabilityMap = make(map[string]capability.Cap, capability.CAP_LAST_CAP+1)
-	for _, c := range capability.List() {
-		if c > capability.CAP_LAST_CAP {
-			continue
-		}
+	list, err := capability.ListSupported()
+	if err != nil {
+		return
+	}
+	capabilityMap = make(map[string]capability.Cap, len(list))
+	for _, c := range list {
 		capabilityMap["CAP_"+strings.ToUpper(c.String())] = c
 	}
 }
 
 // KnownCapabilities returns the list of the known capabilities.
 // Used by `runc features`.
 func KnownCapabilities() []string {
-	list := capability.List()
+	list, err := capability.ListSupported()
+	if err != nil {
+		return nil
+	}
```

Comment:
This is not right.

`runc features` generally show all *known* features, flags etc., even those not supported by the current kernel, platform etc.

I was thinking about adding `--supported` flags but never got around to it.